js/exporter/jspdf/export.js

```diff
@@ -0,0 +1,178 @@
+                        const pdfCell = {
+                            content: cellData.value,
+                            styles: {}
+                        };
+
+                        this.assignCellStyle(pdfCell, gridCell, columns[cellIndex], cellStyle);
+
+                        rowType = rowType || gridCell.rowType;
+                        if(rowType === 'header') {
+                            if(pdfCell.content !== '') { row.push(pdfCell); }
+                        } else {
+                            row.push(pdfCell);
+                        }
+                    }
+
+                    if(rowType === 'header') { autoTableOptions.head.push(row); }
+                    if(rowType === 'data') { autoTableOptions.body.push(row); }
+                }
+
+                jsPDFDocument.autoTable(autoTableOptions);
+
+                resolve(autoTableOptions);
+            }).always(() => {
+
+            });
+        });
+    },
+
+    assignCellStyle: function(pdfCell, gridCell, column, cellStyle) {
+        if(gridCell.rowType === 'header') {
+            // eslint-disable-next-line spellcheck/spell-checker
+            if(column.alignment) { pdfCell.styles.halign = column.alignment; }
+        } else {
+            if(cellStyle) {
+                // eslint-disable-next-line spellcheck/spell-checker
+                if(cellStyle.alignment) { pdfCell.styles.halign = cellStyle.alignment; }
+                if(cellStyle.bold) { pdfCell.styles.fontStyle = 'bold'; }
+                if(cellStyle.wrapText) { pdfCell.styles.cellWidth = 'wrap'; }
+                if(cellStyle.dataType === 'date') { pdfCell.content = dateLocalization.format(new Date(pdfCell.content), cellStyle.format); }
+            }
+        }
+    },
+
+    setColumnWidths: function(autoTableOptions, columns, keepColumnWidths) {
+        const columnStyles = autoTableOptions.columnStyles;
+
+        for(let i = 0; i < columns.length; i++) {
+            columnStyles[i] = columnStyles[i] || {};
+
+            const columnWidth = columns[i].gridColumn.width;
+            if(keepColumnWidths && (typeof columnWidth === 'number') && isFinite(columnWidth)) {
+                columnStyles[i].cellWidth = this.convertUnitToPoints(columnWidth, 'px');
+            } else {
+                columnStyles[i].cellWidth = 'auto';
+            }
+        }
+    },
+
+    getRelativeProportionsToPt: function(unitType) {
+        let k = 1;
+        if(unitType === 'mm') {
+            k = 72 / 25.4;
+        } else if(unitType === 'cm') {
+            k = 72 / 2.54;
+        } else if(unitType === 'in') {
+            k = 72;
+        } else if(unitType === 'px') {
+            k = 96 / 72;
+        } else if(unitType === 'pc') {
+            k = 12;
+        } else if(unitType === 'em') {
+            k = 12;
+        } else if(unitType === 'ex') {
+            k = 6;
+        }
+        return k;
+    },
+
+    convertUnitToPoints: function(value, unitType) {
```

Comment:
There are these calls:
1. autoTableOptions.tableWidth = this.convertUnitToPoints(tableWidth, 'px');
2. columnStyles[i].cellWidth = this.convertUnitToPoints(columnWidth, 'px');

Let's reduce it to 'convertPixelsToPoints' and keep 'px' only conversion:

if(unitType === 'px') {
    k = 96 / 72;
}

Reply:
It is a scale parameter for conversion from 'px' to 'pt'.
72 is a points per inch
96 is a pixels per inch
```
px = 96 / 72 * pt
px = 4 / 3 * pt
pt = 3 / 4 * px
```
I will reduce these methods to the 'convertPixelsToPoints' method.